pom.xml

```diff
@@ -107,6 +100,36 @@
                 </configuration>
             </plugin>
         </plugins>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-resources-plugin</artifactId>
+                    <version>3.1.0</version>
+                    <configuration>
+                        <escapeString>\</escapeString>
+                        <encoding>UTF-8</encoding>
+                    </configuration>
+                </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-archetype-plugin</artifactId>
+                    <version>3.1.3-SNAPSHOT</version>
+                </plugin>
+            </plugins>
+        </pluginManagement>
     </build>
 
+    <!-- Remove once maven-archetype-plugin get released-->
```

Comment:
Even with this section present I had issues building the archetypes project (within NetBeans 11.1).  Maybe it was a transient issue but it might be better to stick with 3.1.2 as it seems to build fine with that version anyway ?